src/test/java/HomeWork4/tests/CheckoutInfoTests.java

```diff
@@ -0,0 +1,19 @@
+package HomeWork4.tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CheckoutInfoTests extends BaseTest {
+    @Test
+    public void CheckoutInfoTest ()  {
+        String productName = "Sauce Labs Onesie";
+        String productName2 = "Sauce Labs Backpack";
+        loginPage.login("standard_user", "secret_sauce");
+        productsPage.clickAddToCartButton(productName);
+        productsPage.clickAddToCartButton(productName2);
+        productsPage.clickYourCart();
+        yourCartPage.clickCheckout();
+        checkoutInfo.setYourInformation("Jonh", "Smith", "222160");
+        Assert.assertEquals(checkoutOverviewPage.getCheckoutOverview(), "Checkout: Overview");
```

Comment:
What is this test about? It verifies that there's a "Checkout: Overview" text on checkout page? - than it's not a functional test, please rework

Reply:
At least rename  the method from getCheckoutOverview() -> getPageTitle()